### PR TITLE
Setup for linters and website files.

### PR DESCRIPTION
**Pull request by lincoln1883**
Add linters and gitignore files as best practice.
Add a copy of files from an existing project for this exercise.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute Page</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Diego Armando Maradona</h1>
+      <div id="img-div">
```
Comment (lincoln1883) [inline index.html:15]: Hi, lincoln I would like to suggest using a section tag instead of a div for semantics purposes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <img id="image" src="https://i.pinimg.com/originals/79/85/08/7985080c1431619fa89314e11aff86b3.png" alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+        <figcaption id="img-caption"> 
+          This image represent Diago Armando Maradona making some magic things with the ball!
+        </figcaption>
+      <div src="js/index.js"></div>
```
Comment (lincoln1883) [inline index.html:20]: Great going so far lincoln, but could it be possible to omit this div, I don't believe this project is using javascript.

**Review by lincoln1883 — COMMENTED** [review]
# Changes required ♻️ 

Hi Lincoln great job so far, please check my comments below and make the required changes.
 
Submit for re-review when you have done so 👏🏾.

Thanks 👍🏾

File: index.html (hunk 1 above)
Comment (divinecharlotte) [reply index.html:15]: - [ ] kindly consider removing that question mark to the end to make your sentence more clear

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @lincoln1883  



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Awesome beginning :heavy_check_mark: 


## Required Changes ♻️

- [ ] kindly consider using some emojis to make the code review more human-friendly 👽🤖.


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi @lincoln1883 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

✔️ Perform code review
✔️ Use the inline comment feature
✔️ Comments are polite

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (abdulhamiid) [reply index.html:20]: **[OPTIONAL]**
You can consider using a couple of emojis and/or gifs in the review.

They make the code review more human-friendly 👽🤖.

File: index.html (hunk 1 above)
Comment (abdulhamiid) [reply index.html:15]: **[OPTIONAL]**
You can consider using a couple of emojis and/or gifs in the review.

They make the code review more human-friendly 👽🤖.

**Pull request merged**